Runtime/Data/ColorPalette.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    [Serializable]
+    public struct ColorPalette
+    {
+        public AssetType assetType;
+        public string[] hexColors;
+
+        public ColorPalette(string name, string[] colorHex)
+        {
+            hexColors = new string[colorHex.Length];
+            for (var i = 0; i < colorHex.Length; i++)
+            {
+                hexColors[i] = colorHex[i];
+            }
+            assetType = ColorResponseHandler.KeyToAssetType(name);
+        }
+    }
```

Comment:
Wouldn't just passing asset type and hexColor would be better? Also I am not sure why the array is filled like that.
```suggestion
         public ColorPalette(AssetType assetType, string[] hexColors)
         {
            this.hexColors = hexColors;
            this.assetType = assetType;
         }
    }
```

Reply:
yes good point!
Updated